docs/ubuntu-installation-guide.md

```diff
@@ -5,7 +5,7 @@ Note:
 If you are installing on a system that already has Clear Containers 2.x
 installed, first read [the upgrading document](upgrading.md).
 
-Clear Containers **3.0** is available for Ubuntu\* **16.04** , **16.10** and **17.04**.
+Clear Containers **3.0** is available for Ubuntu\* **16.04** , **17.04** and **17.10**.
```

Comment:
Only suggest adding the final series comma :) for consistency per the style guide: Clear Containers **3.0** is available for Ubuntu\* **16.04** , **17.04**, and **17.10**.